cli/cmd/inject.go

```diff
@@ -165,6 +172,8 @@ func (rt resourceTransformerInject) transform(bytes []byte) ([]byte, []inject.Re
 	}
 	log.Infof("patch generated for: %s", report.ResName())
 	log.Debugf("patch: %s", patchJSON)
+
```

Comment:
Follow up to the previous comment. The patch generated is an empty array. Should this be of any concern? Am I missing something?